sys-fs/cryptsetup/cryptsetup-1.7.4.ebuild

```diff
@@ -20,8 +20,7 @@ CRYPTO_BACKENDS="+gcrypt kernel nettle openssl"
 # and it's missing ripemd160 support so it can't provide full backward compatibility
 IUSE="${CRYPTO_BACKENDS} libressl nls pwquality python reencrypt static static-libs udev urandom"
 REQUIRED_USE="^^ ( ${CRYPTO_BACKENDS//+/} )
-	python? ( ${PYTHON_REQUIRED_USE} )
-	static? ( !gcrypt )" #496612
```

Comment:
Probably you could put a note in the bug referenced that the problem doesn't exist anymore (if it does not).

Reply:
To be extra safe the line below 

```
gcrypt? ( dev-libs/libgcrypt:0=[static-libs(+)] )
```

could get a `>=` a known working version.

Reply:
The earliest version I can confirm this worked with is dev-libs/libgcrypt-1.5.4-r1.  The oldest version in portage is dev-libs/libgcrypt-1.5.6, does it still need a >= just to be safe?

Reply:
@negge yes, what if a user has dev-libs/libgcrypt-1.5.3 installed on their system? You want to make them upgrade to 1.5.4-r1 at least.

Reply:
@robbat2 If they have libgcrypt-1.5.3 installed (which may work) and cryptsetup[static,-gcrypt] then we know that changing cryptsetup to allow both static and gcrypt will not break their setup because they had to USE=-gcrypt to get cryptsetup to install statically.

This should not be an issue for existing installations I don't think

Reply:
It will work for now, because they still have `cryptsetup[static,-gcrypt]`; if the `-gcrypt` gets lost (say accidently), and they still have the old gcrypt, which happened to be broken, it would be better to force the upgrade (to at least 1.5.4-r1 that you confirmed)